spec/avram/bulk_upsert_spec.cr

```diff
@@ -0,0 +1,19 @@
+require "../spec_helper"
```

Comment:
Delete this file

Reply:
I think we may want a few small tests around this.